### ISSUE TITLE
Update to 'Windows 11, version 24H2' not working!

### ISSUE THREAD

**Issue by Gobblemanz** [valinet/ExplorerPatcher#3994]
My laptop tried to update to 'Windows 11, version 24H2' but when  it tried it goes to 14% then has a Windows 11 update popup come up telling me I have an issue with 'ExplorerPatcher'. It doesn't do anything when I refresh and when i exit out of the popup it just cancels the update. its been about a week n nothin changed.
![image](https://github.com/user-attachments/assets/d3ffc562-1946-420e-9f6f-c42205d281f1)
![image](https://github.com/user-attachments/assets/afcd85f9-9914-4b55-bed9-2d278933374c)

**Comment by pyrates999** [comment]
Uninstall EP, then you can upgrade to 24h2.  Install EP after reading the release notes here: https://github.com/valinet/ExplorerPatcher/releases

**Comment by Panzimy** [comment]
Yeah, the OS with EP can't be updated as @pyrates999 said 
> Uninstall EP, then you can upgrade to 24h2. Install EP after reading the release notes here: https://github.com/valinet/ExplorerPatcher/releases
>
**BUT** before uninstalling find in EP export current settings a export settings somewhere, uninstall EP and after update only install EP and after installation import settings

**Comment by pyrates999** [comment]
I prefer setting up the settings again in case there are issues with importing the settings.

**Comment by mustafatoktas** [comment]
I think this situation is very funny. Why is Microsoft so afraid of Explorer Patcher? 😄

**Comment by Holiwood43** [comment]
Dear Gobblemanz,  I faced the exact same issues.  What I did was to change all figures in Explorer Patcher back to Windows 11 and afterwards I was able to deinstall the EP Patcher.  Some useless Notification popped up but after done I was able to gat the new Update 24H2.  Before I started to  chnage the Figures from Win 10 Style... I exported the settings for easy return, which I hope will work, LetÄs see. First of all. I got the Update on my PC.  Have a good Time :)
